tools/isobuild/compiler-plugin.js

```diff
@@ -783,18 +783,16 @@ class ResourceSlot {
         // stub, so setting .implicit marks the resource as disposable.
       }).implicit = true;
 
-      // TODO[FIBERS]: Look into this. We probably don't want addStylesheet
-      // to be async, and I'm also not sure the old behavior here is what we wanted
-      //
-      // if (! cssResource.lazy &&s
-      //     ! Buffer.isBuffer(cssResource.data)) {
-      //   // If there was an error processing this file, cssResource.data
-      //   // will not be a Buffer, and accessing cssResource.data here
-      //   // should cause the error to be reported via inputFile.error.
-      //   return;
-      // }
-
-      this.outputResources.push(cssResource);
+      this.outputResources.push(async () => {
+        if (! cssResource.lazy &&
+          ! Buffer.isBuffer(await cssResource.data)) {
```

Comment:
To add some context to why this code was commented out, it being async is only part of the issue. The main problem is the code doesn't seem to work correctly - it either does nothing, or if a build plugin reported an error it will crash Meteor. I think we should leave this code disabled until we can understand what it is trying to do and figure out a better way to achieve it. It would also be worth checking if this worked correctly in Meteor 2, and why it worked there.